profiles/arch/amd64/package.use.mask

```diff
@@ -17,6 +17,10 @@
 
 #--- END OF EXAMPLES ---
 
+# Nicolas Werner <nicolas.werner@hotmail.de> (2021-09-05)
+# Qt5 packages don't support multilib, so disable it for this plugin.
+media-plugins/gst-plugins-qt5 abi_x86_32
```

Comment:
I feel a bit uneasy about an exception here, when we've had it all work for everything else. I guess it's a bit special, but maybe it'd be possible to not add the multilib stuff to it in the first place. However that's probably too much eclass churn..

Reply:
I think this is the easiest solution for now. If we made the multilib optional in the eclass and only had one consumer using that, it would not get much testing and add a bit of complexity. I'd argue changing the eclass, if there is a second or third package with this issue makes the most sense?